inst/components/sidebar.scss

```diff
@@ -110,14 +110,14 @@ $bslib-sidebar-border: var(--bs-card-border-width, #{$card-border-width}) solid
     display: inline-flex;
     align-items: center;
     position: absolute;
-    right: - $bslib-sidebar-icon-size;
+    right: calc(-#{$bslib-sidebar-icon-size} - var(--bs-card-border-width, #{$card-border-width}));
```

Comment:
Let's make sure we don't have the same issue on mobile or `sidebar(position="right")`